src/emails/EmailService.ts

```diff
@@ -0,0 +1,33 @@
+import { User, UserModel } from './../schemas/User';
+
+const sendmail = require('sendmail')();
+
+export class EmailService {
+
+	sendMail(from: string, to: string, subject: string, message: string): Promise<any> {
```

Comment:
- Better another name for the function, because is very similar to the library, because only differs in one character "sendmail - sendMail", and it could be confused.